elastic/kibana
# [Alerting] Keep the number_of_scheduled_actions in event log

fixes: #127448

## Summary

We keep the `number_of_triggered_actions` in event log but this statistic could be misleading after the max executable actions circuit breaker feature. Therefore we would like to keep `number_of_scheduled_actions` as well. So we can compare  how many actions have been scheduled and how many of them were triggered/skipped.

## To Verify:
Create a rule that has multiple actions.
Set the `xpack.alerting.rules.execution.actions.max` in kibana.yml lesser than the number of actions you have created in your rule.
Let your rule run.
Check the `number_of_triggered_actions` and `number_of_scheduled_actions` under metrics object in your `execute` event log.
